gallery/plot_visualization_utils.py

```diff
@@ -38,10 +39,14 @@ def show(imgs):
 
 from torchvision.utils import make_grid
 from torchvision.io import read_image
+from torchvision.transforms.functional import convert_image_dtype
 from pathlib import Path
 
+
 dog1_int = read_image(str(Path('assets') / 'dog1.jpg'))
+dog1 = convert_image_dtype(dog1_int, dtype=torch.float)
```

Comment:
We still need the int images for make_grid but quite frankly I'm tempted to remove `make_grid` from this example